recipes/edlib/all/patches/cmake-install-and-export-symbols.patch

```diff
@@ -0,0 +1,114 @@
+--- a/CMakeLists.txt
```

Comment:
Looks like a patch ready for upstream 💪

Reply:
https://github.com/Martinsos/edlib/pull/167